### PR TITLE
Sort repositories list by name

### PR DESCRIPTION
**Pull request by eugene-bright**
Sorting tool is also provided.

### CODE REVIEW

**Comment by eugene-bright** [comment]
I can't see any reasons for checking module name in simple script that is not designed to be imported.
I will rework code to be reusable if you wish.

**Comment by eugene-bright** [comment]
Sorting is case insensitive now.
Code smells much better.

File: bin/sort_repositories.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    sort_repo_nodes(repo_nodes)
+    reinsert_repo_nodes(
+            repositories_node,
+            repo_nodes)
+
+    vim_modelines_fixup(repositories_node)
+
+    sys.stdout.write(
+        format_xml(repo_etree))
+
```
Comment (mgorny) [inline bin/sort_repositories.py]: Stray empty line ;-).

File: bin/sort_repositories.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+
```
Comment (mgorny) [inline bin/sort_repositories.py]: I'm sorry that I didn't notice this before. Could you please choose a license and add a (short) copyright statement? Any FSF/OSI-approved license would work, I think, with preference of GPL or BSD variants. If you really don't want to hold copyright, then please use CC0 (public domain equivalent).

Comment (eugene-bright) [reply bin/sort_repositories.py]: Other files do not contain any license information also.
I personally prefer [GPL-3.0-or-later](https://spdx.org/licenses/GPL-3.0-or-later.html).
Could we add project-wide license manifesto (LICENSE file at the root directory)?

Comment (eugene-bright) [reply bin/sort_repositories.py]: I've added license information to the script to accelerate this pull request.

**Comment by eugene-bright** [comment]
Let me know if more corrections are needed.

File: bin/sort_repositories.py (hunk 2 above)
Comment (mgorny) [reply bin/sort_repositories.py]: This repository is mostly concerned about data files that are non-copyrightable. Only the scripts are copyrightable, and there's no reason to force a single license for them.

That said, GPL explicitly requires the full license notice in every file, and a copy of GPL in the repo.

Comment (eugene-bright) [reply bin/sort_repositories.py]: Looks like all requirements are met now.

**Comment by mgorny** [comment]
Thanks a lot!

**Comment by mgorny** [comment]
FTR, I've squashed your commits a bit more and fixed one whitespace error in the script (empty indented line).

**Comment by eugene-bright** [comment]
In the docstring empty indented line in a part of docstring, not just meaningless indentation.
But it seems to be processed equally by `textwrap.dedent`, so that's ok.

**Pull request closed without merging**